styles/duolingo/catppuccin.user.css

```diff
@@ -0,0 +1,362 @@
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+    @brightness: if(@lookup = latte, 100%, 0);
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    /* Start styling your website here: */
+    background-color: @base;
+
+    /* If you need to specify an overwrite specific to a flavor
+    *  (e.g. only when the flavor is set to latte), you can use a `when` guard clause. */
+    & when (@lookup = latte) {
+      /* ... */
+    }
+    /* You can also negate the statement above by using
+    *  a `when not` guard clause. */
+    & when not (@lookup = latte) {
+      /* ... */
+    }
+
+    ._3HqVu,
+    .uGhFr {
+      filter: brightness(@brightness);
+    }
+
+    .ydKok,
+    .Ni_YG {
+      color: @text;
+    }
+
+    * {
+      --color-snow: #rgbify(@base) [];
+      --color-snow-always-light: #rgbify(@base) [];
+      --color-snow-always-dark: #rgbify(@mantle) [];
+      --color-polar: #rgbify(@base) [];
+      --color-swan: #rgbify(@surface1) [];
+      --color-swan-always-light: #rgbify(@rosewater) [];
+      --color-swan-always-dark: #rgbify(@surface1) [];
+      --color-hare: #rgbify(@surface2) [];
+      --color-hare-always-light: #rgbify(@subtext0) [];
+      --color-wolf: #rgbify(@text) [];
+      --color-eel: #rgbify(@rosewater) [];
+      --color-squid: #rgbify(@rosewater) [];
+      --color-walking-fish: #rgbify(@base) [];
+      --color-flamingo: #rgbify(@surface2) [];
+      --color-pig: #rgbify(@maroon) [];
+      --color-crab: #rgbify(@red) [];
+      --color-cardinal: #rgbify(@red) [];
+      --color-fire-ant: #rgbify(@red) [];
+      --color-canary: #rgbify(@base) [];
+      --color-duck: #rgbify(@peach) [];
+      --color-bee: #rgbify(@peach) [];
+      --color-bee-always-dark: #rgbify(@peach) [];
+      --color-lion: #rgbify(@peach) [];
+      --color-fox: #rgbify(@peach) [];
+      --color-cheetah: #rgbify(@base) [];
+      --color-monkey: #rgbify(@peach) [];
+      --color-camel: #rgbify(@peach) [];
+      --color-guinea-pig: #rgbify(@peach) [];
+      --color-grizzly: #rgbify(@overlay0) [];
+      --color-sea-sponge: #rgbify(@base) [];
+      --color-turtle: #rgbify(@surface1) [];
+      --color-owl: #rgbify(@green) [];
+      --color-tree-frog: #rgbify(@overlay0) [];
+      --color-peacock: #rgbify(@sapphire) [];
+      --color-iguana: #rgbify(@base) [];
+      --color-anchovy: #rgbify(@text) [];
+      --color-beluga: #rgbify(@text) [];
+      --color-moon-jelly: #rgbify(@sky) [];
+      --color-blue-jay: #rgbify(@overlay0) [];
+      --color-macaw: #rgbify(@sapphire) [];
```

Comment:
Colors like this that are sapphire should probably be accent, since Duolingo uses this blueish/sapphire color as their own (secondary) accent. You could probably just do a find and replace across this whole file and do them all at once.